azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerPipelineTest.java

```diff
@@ -84,7 +82,7 @@ public FlowRunnerPipelineTest() {
   @Before
   public void setUp() throws Exception {
     System.out.println("Create temp dir");
-    this.workingDir = new File("_AzkabanTestDir_" + System.currentTimeMillis());
+    this.workingDir = new File("build/tmp/_AzkabanTestDir_" + System.currentTimeMillis());
```

Comment:
Is it better to use junit's temp directory rule?
https://garygregory.wordpress.com/2010/01/20/junit-tip-use-rules-to-manage-temporary-files-and-folders/